src/StatusQ/Controls/StatusChatInfoButton.qml

```diff
@@ -46,191 +40,158 @@ Rectangle {
+                        onClicked: root.unmute()
+                    }
 
-        Item {
-            id: statusChatInfoButtonSubTitle
-            anchors.left: statusChatInfoButtonTitle.left
-            anchors.top: statusChatInfoButtonTitle.bottom
-            visible: !!statusChatInfoButton.subTitle || statusChatInfoButton.pinnedMessagesCount > 0
-            height: visible ? chatType.height : 0
-            width: Math.min(parent.width - statusChatInfoButtonTitle.anchors.leftMargin
-                            - identicon.width - identicon.anchors.leftMargin - 8,
-                            implicitWidth)
-
-            implicitWidth: chatType.implicitWidth + pinIconDelta + 8
-
-
-            property real pinIconDelta: pinIcon.visible ? pinIcon.width + pinIcon.anchors.leftMargin
-                                                          + divider.width + divider.anchors.leftMargin
-                                                        : 0
-
-            StatusSelectableText {
-                id: chatType
-                text: statusChatInfoButton.subTitle
-                color: Theme.palette.baseColor1
-                font.pixelSize: 12
-                width: Math.min(parent.width - (pinIcon.visible ? divider.width + divider.anchors.leftMargin + pinIcon.width + pinIcon.anchors.leftMargin : 0),
-                                implicitWidth)
-                onLinkActivated: statusChatInfoButton.linkActivated(link)
+                    StatusToolTip {
+                        id: statusToolTip
+                        text: qsTr("Unmute")
+                        visible: mutedIconSensor.containsMouse
+                        orientation: StatusToolTip.Orientation.Bottom
+                        y: parent.height + 12
+                    }
+                }
             }
 
-            Rectangle {
-                id: divider
-                height: 12
-                width: 1
-                color: Theme.palette.directColor7
-                anchors.left: chatType.right
-                anchors.leftMargin: 4
-                anchors.verticalCenter: chatType.verticalCenter
-                visible: !!chatType.text && pinIcon.visible
-            }
+            // subtitle
+            RowLayout {
+                id: subtitleRow
+                Layout.fillWidth: true
+                Layout.alignment: Qt.AlignLeading | Qt.AlignTop
+                spacing: 0
+
+                StatusSelectableText {
+                    Layout.fillWidth: implicitWidth + separator.width + pinIcon.width + pinText.width > subtitleRow.width
+                    text: root.subTitle
+                    color: Theme.palette.baseColor1
+                    font.pixelSize: 12
+                    onLinkActivated: root.linkActivated(link)
+                }
 
-            StatusIcon {
-                id: pinIcon
+                Rectangle {
+                    id: separator
+                    Layout.preferredHeight: 12
```

Comment:
Some time ago we discussed that orders of QML properties should be from parent to child and attached properties should be last

Reply:
OK, is that documented somewhere? Just wondering because the usual practice is placing them first